bin/main.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../lib/game'
+require_relative '../lib/user'
+puts 'Hello welcome to the Tic Tac Toe Game'
+userx = User.new('X')
+userx.set_name
+usery = User.new('O')
+usery.set_name
+puts '--------------------------------------'
+puts 'This our board'
```

Comment:
Here, ask the user's name and read here too. Then, use them to instantiate the `User`